syz-cluster/pkg/db/migrations/1_initialize.up.sql

```diff
@@ -69,6 +69,7 @@ CREATE TABLE SessionTests (
     Result STRING(36) NOT NULL,
     BaseBuildID STRING(36),
     PatchedBuildID STRING(36),
+    LogURI STRING(256) NOT NULL,
```

Comment:
256 looks too short for the log.
What do you expect the actual URI to look like?

Reply:
In production it will be a path to a GCS object, e.g. `gcs://bucket-name/some-random-string`. 256 bytes should be more than enough for such URIs.